extension/scripts/features/gym-graph/ttGymGraph.js

```diff
@@ -127,7 +127,14 @@
 
 			const context = canvas.getContext("2d");
 
-			createChart();
+			const gymChart = createChart();
+			darkModeObserver.addListener(darkMode => {

```

Comment:
Isn't it possible to do this purely with css? That way the dark mode observer is not needed.